cognite/client/_api/relationships.py

```diff
@@ -149,7 +165,33 @@ def retrieve_multiple(self, external_ids: List[str]) -> RelationshipList:
                 >>> res = c.relationships.retrieve_multiple(external_ids=["abc", "def"])
         """
         utils._auxiliary.assert_type(external_ids, "external_id", [List], allow_none=False)
-        return self._retrieve_multiple(external_ids=external_ids, wrap_ids=True)
+        relationships = self._retrieve_multiple(
+            external_ids=external_ids, wrap_ids=True, other_params={"fetchResources": fetch_resources}
+        )
+        [RelationshipsAPI._convert_resources(r) for r in relationships]
+        return relationships
+
+    @staticmethod
+    def _convert_resources(relationship: Relationship):
+        if relationship is not None:
+            if relationship.source is not None:
+                relationship.source = RelationshipsAPI._convert_resource(relationship.source, relationship.source_type)
+            if relationship.target is not None:
+                relationship.target = RelationshipsAPI._convert_resource(relationship.target, relationship.target_type)
+
+    @staticmethod
+    def _convert_resource(resource: Dict, resource_type: str):
+        resource = {utils._auxiliary.to_snake_case(key): value for key, value in resource.items()}
+        if resource_type == "timeSeries":
```

Comment:
This if-else ladder could be changed to something cleaner but I'm not exactly sure what. maybe a dict?